src/main/java/adubbz/nx/loader/common/MemoryBlockHelper.java

```diff
@@ -67,7 +67,7 @@ private void addUniqueSection(String name, long addressOffset, long offset, long
             newBlockName = name + "." + nameCounter; 
         }
         
-        Msg.info(this, "Adding unique section " + newBlockName + " from " + startAddr.toString() + " to " + endAddr.toString());
+        Msg.info(this, "Adding unique section " + newBlockName + " from " + startAddr + " to " + endAddr.toString());
```

Comment:
Did you miss that ".toString()`` or did you leave it in on purpose?

Reply:
I missed it, thank you! :D